### PR TITLE
Add meta files from maintainer-quality-tools

### PR DESCRIPTION
**Pull request by bwrsandman**
Ignore OSX Files temp files in source
Add License file
Add travis and coverage badges

### CODE REVIEW

**Comment by yvaucher** [comment]
Can you change badge to svg ?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install geojson Shapely
+  - sudo apt-get -q -y install postgis
+
+script:
+  - travis_run_tests
+
+after_success:
+  coveralls
```
Comment (pedrobaeza) [inline .travis.yml]: Put `travis_after_tests_success`

**Comment by nbessi** [comment]
need to be rebased on https://github.com/OCA/geospatial/pull/31

**Comment by bwrsandman** [comment]
#31 is a v8, I can't do that.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="6.1" LINT_CHECK="1"
```
Comment (pedrobaeza) [inline .travis.yml:7]: You can put globals

**Comment by nbessi** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**